manager/state/raft/membership/cluster.go

```diff
@@ -166,18 +166,14 @@ func (c *Cluster) ValidateConfigurationChange(cc raftpb.ConfChange) error {
 func (c *Cluster) CanRemoveMember(from uint64, id uint64) bool {
 	members := c.Members()
 
-	nmembers := 0
+	nmembers := 1
```

Comment:
How about `nmembers := len(members) + 1`? And probably rename it too, since it's no longer the number of members.